src/lib/core/README.md

```diff
@@ -1,6 +1,10 @@
-# esri-map
+# EsriModuleProvider
+
+`EsriModuleProvider`  a component which abstracts the esri loader component using [ArcGIS API for JavaScript v4.6](https://developers.arcgis.com/javascript/)
```

Comment:
component is not the correct term to use here. In Angular the term component is something visual with a template that contributes to the DOM for your application. service or provider would be more appropriate terms to use here.